microsoft/testsuites/network/sriov.py

```diff
@@ -542,6 +544,158 @@ def sriov_ethtool_offload_setting(self, environment: Environment) -> None:
         )
         assert_that(iperf_log).does_not_contain("error")
 
+    @TestCaseMetadata(
+        description="""
+        This case is to verify interrupts count increased after network traiffic
+         went through the VFs.
+
+        Steps,
+        1. Start iperf3 on server node.
+        2. Get initial interrupts sum per irq and cpu number on client node.
+        3. Start iperf3 for 5 minutes with 128 threads on client node.
+        4. Get final interrupts sum per irq number on client node.
+        5. Compare interrupts changes, expected to see interrupts increased.
+        6. Get final interrupts sum per cpu on client node.
+        7. Compare interrupts changes, expected to see interrupts increased.
+        """,
+        priority=2,
+        timeout=1200,
+        requirement=node_requirement(
+            node=schema.NodeSpace(
+                node_count=2,
+                core_count=search_space.IntRange(min=8, max=16),
```

Comment:
What's the reason to limit core count between 8 and 16? If there is any reason, add it in test spec.

Reply:
will add later

Reply:
updated

Reply:
The description can be clear more. Explain explicitly after the "too many and too few", like "so it chooses cpu count between 8 to 16.

Reply:
I am not 100% sure for this range, my understanding, if core is too few, the test is useless, I tested when core is many at least >=32, some cpu are idle, the case will fail (I will add one more case for this situation, to update the target CPU)

Reply:
Anyway, we should document what you thought. It will be much more confusing, if we hide the test design idea.

Reply:
updated